stdpopsim/catalog/ApiMel/genome_data.py

```diff
@@ -3,22 +3,24 @@
     "assembly_accession": "GCA_003254395.2",
     "assembly_name": "Amel_HAv3.1",
     "chromosomes": {
```

Comment:
Oh, this is why Codecov is complaining (no more synonyms, so this bit of code in `ApiMel/species.py` is not needed)
![Screenshot from 2025-01-13 21-54-18](https://github.com/user-attachments/assets/55f3c7c6-6c06-4be8-8b3a-69ca69759c4e)